alternator/server.hh

```diff
@@ -24,16 +24,19 @@
 #include "alternator/executor.hh"
 #include <seastar/core/future.hh>
 #include <seastar/http/httpd.hh>
+#include <seastar/net/tls.hh>
+#include <optional>
 
 namespace alternator {
 
 class server {
     seastar::httpd::http_server_control _control;
+    seastar::httpd::http_server_control _tls_control;
```

Comment:
I allowed it, because I'm not sure. Perhaps we want to serve HTTPS, but with a possibility to degrade to HTTP if the client wishes. I'm open to suggestions though.

Reply:
I don't see the use-case. If one wants HTTPS falling back to HTTP is unacceptable. But I'll let others with more operation experience chime in.

Reply:
I think Nadav also had allowing both in mind, because the original issue mentions `allow both parameters, or just one, or none at all, to be used` when talking about the tls port parameter, so I went with the most inclusive option. I'll leave it for discussion though, changing the code to support just one port parameter + a tls boolean is easy to migrate to.